k8s/base/exporter.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
should we use `cronJob` for exporter instead of `Deployment`?

Reply:
okay, makes sense to leave it as is then

Reply:
curious that if we use `kind: Deployment` would it make the exporter pod to run constantly even though it wouldn't be doing anything?

Reply:
it would. let's revisit the `cronJob` option in the next basin iteration... sounds like a nice setup but would require code changes as bruno mentioned

Reply:
> curious that if we use `kind: Deployment` would it make the exporter pod to run constantly even though it wouldn't be doing anything?

good point!